extra_tests/snippets/stdlib_io.py

```
@@ -84,3 +84,30 @@ def write(self, data):
 with assert_raises(AttributeError):
     textio.writelines(["x"])
+
+textio = TextIOWrapper(BytesIO())
+
+for invalid_chunk_size in (0, -1, 2**100):
+    try:
+        textio._CHUNK_SIZE = invalid_chunk_size
+    except ValueError:
+        pass
+    else:
+        assert False, f"expected ValueError for {invalid_chunk_size!r}"
```

Ruff (0.15.20)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)